From 8f6304184d2a8df0af4aacf836a2a3d20383fa80 Mon Sep 17 00:00:00 2001
From: Your Name <1446775+jacaru@users.noreply.github.com>
Date: Sat, 25 Mar 2023 19:24:48 +0100
Subject: [PATCH 1/4] Enable trailing slashes

By default, next.js redirects trailing slash urls to non trailing slash.
In the worst case, this causes redirect loops in the most typical reverse
proxy configurations. In the best, it just makes your life harder when
setting up your reverse proxy handlers. This type of redirect is better
handled on the proxy itself and I find having trailing slashes enabled a
better overall default.
---
 next.config.js | 1 +
 1 file changed, 1 insertion(+)

diff --git a/next.config.js b/next.config.js
index dae78ca0..e2efad75 100644
--- a/next.config.js
+++ b/next.config.js
@@ -2,6 +2,7 @@ const { i18n } = require("./next-i18next.config");
 
 /** @type {import('next').NextConfig} */
 const nextConfig = {
+  trailingSlash: true,
   reactStrictMode: true,
   output: "standalone",
   swcMinify: false,
-- 
2.30.2

